source/whats-new.txt

```diff
@@ -28,8 +28,31 @@ Learn what's new in:
 What's New in 4.6
 -----------------
 
-See `v4.6.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.6.0>`__
-on GitHub.
+New features of the 4.6 Node.js driver release include:
+
+TypeScript: ``ChangeStreamDocument``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Ability to define your own custom type for the top level document
+  returned in a ``change`` event.
+- Improvements on the type information of the default
```

Comment:
Suggestion: I'm not able to identify more than one improvement listed. Instead, why not just describe the improvement, e.g (I'm not sure this is correct, would need to read the notes).:
"The ChangeStreamDocument now defaults to the option to. union all the possible event types"